### ISSUE TITLE
False positive with link-image-style rule (MD054) and `{ url_inline: false }`

### ISSUE THREAD

**Issue by regseb** [DavidAnson/markdownlint#1040]
The [_link-image-style_](https://github.com/DavidAnson/markdownlint/blob/b709a2f/doc/md054.md) rule (MD054), with `{ url_inline: false }` config, reports a false-positive issue to this link:

```markdown
[link](https://example.com) 
```

**Comment by DavidAnson** [comment]
OK, it looks like the tests for this scenario are very focused on what conditions should fire the warning and not at all focused on ensuring the text matches the link. My bad.
